### PR TITLE
Add feature discover section

### PR DESCRIPTION
**Pull request by janhendrikstruve**

### CODE REVIEW

**Review by derjubu — APPROVED** [review]
Looks fine to me

File: styles/components/redirection.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.redirection {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-items: center;
+  background-color: #332f3c;
```
Comment (badabam) [inline styles/components/redirection.css:5]: Please consider css variables for all colors.

File: styles/components/redirection.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  margin: 0 0 64px;
+}
+
+.redirection__button--pink {
+  justify-self: right;
+}
+
+.redirection__button--black {
+  justify-self: left;
+}
```
Comment (badabam) [inline styles/components/redirection.css:31]: Class names and properties don't fit together. Better `redirection__button--left` as color does not matter here. Also: why not using the parent element to justify both?

File: styles/components/team.css [hunk 3]
```diff
@@ -22,8 +22,54 @@
...
+  .team__card-text {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+  }
+  .team__list-item {
+    background-color: #f2f2f2;
+  }
+
+  .team__list-item h3 {
```
Comment (badabam) [inline styles/components/team.css:47]: Please fix your BEM. This needs a pure class.

File: styles/components/team.css [hunk 4]
```diff
@@ -22,8 +22,54 @@
...
+    color: #0d0d0d;
+    font-style: italic;
+    font-weight: 700;
+    text-align: left;
+    margin-bottom: 0;
+  }
+
+  .team__list-item span {
```
Comment (badabam) [inline styles/components/team.css:55]: Bad BEM. Please fix this.

File: styles/components/team.css [hunk 5]
```diff
@@ -22,8 +22,54 @@
...
+    color: #332f3c;
+    font-style: italic;
+    font-weight: 700;
+    text-align: left;
+  }
+
+  .team__list-item p {
```
Comment (badabam) [inline styles/components/team.css:62]: Please fix this BEM.

**Review by badabam — COMMENTED** [review]
Please fix your BEM and consider my other comments. You've got this! 💪

**Pull request merged**